app/resources/v1/users.js

```diff
@@ -333,7 +333,6 @@ exports.get = async function (req, res) {
     if (!user) {
       throw new Error(ERRORS.USER_NOT_FOUND)
     }
-
```

Comment:
Since there's no other changes in this file can we remove this line break subtraction from the commit?